sdk/include/psdk/d3d9.h

```diff
@@ -64,7 +64,9 @@
  * #defines and error codes
  */
 #define D3D_SDK_VERSION                         32
+#define D3D9b_SDK_VERSION                       31
```

Comment:
Is this `b` supposed to be lowercase?

Reply:
Yes, because it is used with lowercase `b` in the code: https://github.com/reactos/reactos/blob/e18e7b100c236835ffa6ca06114a5c8551becec5/dll/directx/d3d9/d3d9.c#L83

Reply:
https://docs.microsoft.com/en-us/windows/win32/direct3d9/other-direct3d-constants confirms it.